packages/babel-node/test/fixtures/cli/arguments-categories/options.json

```diff
@@ -8,6 +8,7 @@
     "--expose-gc-as=gc2",
     "--config-file",
     "./config.json",
+    "--extensions=.js,.ts",
```

Comment:
Do we print warning if there are only node options after the script name? E.g.
```
babel-node ./foo.js --expose_gc
```

Reply:
We do not, because it would require us to keep the `v8flags` dependency just for that warning.

However, I'm less worried about that case since when using the Node.js CLI you already have to pass Node-specific options before the script name.

Reply:
I think it is fine to keep or inline the `v8flags` dependency and finally remove it in Babel 9. In Babel 7 the node options after the script name are still passed to node and we align the args parsing only starting from this Babel 8 PR.

There are indeed such usage where node options are after the script name in the wild:
https://github.com/Ocelyn/r3f-next-starter/blob/eaa6ad70409943ca6a6cb05e6e502ae994847a0d/package.json#L12

https://github.com/chairemobilite/evolution/blob/0aecdc436411fa15ef9991a3309c8e2898c2700c/example/demo_survey/package.json#L21

https://github.com/adieuadieu/japan-train-data/blob/0a31ec2665475c3e46da7ffc7786c33be9e7ac05/package.json#L38

Imagine if they upgrade to Babel 8, their app may stop due to insufficient memory size. However, it will be a really confusing error: they might keep increasing the `--max-old-space-size` until they figure out that `--max-old-space-size` after the script name are no longer passed to the node. We could save them lots of time if we just keep `v8flags` for a while and remind them to update their usage.

We can also start printing warnings in Babel 7 if node options after the script name is seen, since they will not be supported in Babel 8.